.gitignore

```diff
@@ -1,5 +1,6 @@
 *~
 .DS_Store
+.idea/
```

Comment:
this should be in your global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore